en/docs/install-and-setup/configuring-identity-server-for-ob.md

```diff
@@ -149,17 +149,50 @@ the consent page.
     single_acc_no_retrieval_validation.enable = false
     ```
 
-14. To enable idempotency support for the Payments API:
+14. Configure the following tags regarding Account consents:
+
+    - `single_acc_no_retrieval_validation.enable`: If enabled, the request permission validation is performed against 
+       the account ID by the bank back end. If disabled, request validation happens against the consent ID by WSO2 Open 
+       Banking Berlin Toolkit.
```

Comment:
Should be corrected like this:
If enabled, the permission validation will be performed against the account ID by the wso2 Open Banking Berlin Toolkit. If disabled, permission validation happens against the consent ID by WSO2 Open Banking Berlin Toolkit. Also when disabled, the account ID validation should be done by the bank backend.

Reply:
Fixed in 7bfa39b2fba54a9ecb39b651ec24854dea85354f